### DISCUSSION TITLE
Ondsel ES as Flatpak

### DISCUSSION THREAD

**Discussion by Tom-Obvious** [Ondsel-Development/FreeCAD#20] [Ideas]
Hi, I really like to work with FreeCAD and I'm curious about the changes of Ondsel ES and its future development.
But, working with Linux and mostly with the Flatpak packaging format, I discovered that Ondsel is only downloadable as AppImage. Why is that? Are there any plans to release Ondsel ES as Flatpak on Flathab as well?

**Comment by kinghat** [comment]
> Hi, I really like to work with FreeCAD and I'm curious about the changes of Ondsel ES and its future development. But, working with Linux and mostly with the Flatpak packaging format, I discovered that Ondsel is only downloadable as AppImage. Why is that? Are there any plans to release Ondsel ES as Flatpak on Flathab as well?

its probably because this is their first release and was the quickest way to get something universal for linux out the door. i was using the flatpak release of freecad and would like a flatpak of ondsel.
